src/mongo/db/dbcommands.cpp

```diff
@@ -367,6 +368,16 @@ class CmdProfile : public Command {
             serverGlobalParams.slowMS = slow.numberInt();
         }
 
+        const BSONElement ratelimitelem = cmdObj["ratelimit"];
+        if (ratelimitelem.isNumber()) {
```

Comment:
Few lines above there is block of code:

        if (slow.isNumber()) {
            serverGlobalParams.slowMS = slow.numberInt();
        }

I just did my part the same way.